service-providers/apple-music-manager/main.py

```diff
@@ -17,6 +18,25 @@
 
 SEND_HEARTBEAT = True
 
+def getCommonFormatSongsFromPlaylists(userToken: str, playlistIDs: List[str]):
+    songs = []
+    for playlistID in playlistIDs:
+        playlist = api.getPlaylistTracks(userToken, playlistID)
+        for track in playlist["data"]:
+            newSong = {}
+
+            newSong["songName"] = track["attributes"]["name"]
+            newSong["artist"] = track["attributes"]["artistName"]
```

Comment:
I think this should be the same format as spotify. "artists": [...] instead of "artist":...

Otherwise, looks good 👍